test/integration/sagemaker/test_experiments.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import time
+
+import pytest
+from sagemaker import utils
+from sagemaker.tensorflow import TensorFlow
+from smexperiments.experiment import Experiment
+from smexperiments.trial import Trial
+from smexperiments.trial_component import TrialComponent
+
+from test.integration import RESOURCE_PATH
+from timeout import timeout
+
+DATA_PATH = os.path.join(RESOURCE_PATH, "mnist")
+SCRIPT_PATH = os.path.join(DATA_PATH, "mnist_gluon_basic_hook_demo.py")
+
+
+def test_training(sagemaker_session, ecr_image, instance_type, framework_version, py_version):
```

Comment:
the last PR build had [the same failure as before](https://github.com/aws/sagemaker-tensorflow-container/pull/286#issuecomment-595884123). I think you need to do something similar to https://github.com/aws/sagemaker-tensorflow-container/blob/master/test/integration/local/test_horovod.py#L22 and put this line in this file:

```python
from test.integration.utils import processor, py_version  # noqa: F401
```